engine.py

```diff
@@ -45,22 +45,24 @@ class BdfCallbacks:
     def __init__(self, registry):
         self.registry = registry
 
-    def check(self, name, property_name):
+    def check(self, name, property_name="condition_met"):
```

Comment:
I temporarily have gave up on using the "condition_met" as a default field name since I am not sure the API is clear enough to inform the user about a default field that he has to set so at the moment I am letting the user to choose the field name to check and set.